ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -95,6 +99,171 @@ def format_locator(locator, string_to_insert):
+        elif vol_binding_mode == "Immediate":
+            logger.info("select Immediate")
+            base_ui_obj.do_click(pvc_loc["immediate"])
+
+    logger.info("Storage Class Provisioner")
+    base_ui_obj.do_click(pvc_loc["provisioner"])
+    if provisioner == "rbd":
+        base_ui_obj.do_click(pvc_loc["rbd-provisioner"])
+    elif provisioner == "cephfs":
+        base_ui_obj.do_click(pvc_loc["cephfs-provisioner"])
+    logger.info("Storage Class Storage Pool")
+    base_ui_obj.do_click(pvc_loc["storage-pool"])
+    base_ui_obj.do_click(pvc_loc["ceph-block-pool"])
+    if encryption:
+        logger.info("Storage Class with Encryption")
+        base_ui_obj.do_click(pvc_loc["encryption"])
+        logger.info("Checking if 'Change connection details' option is available")
+        conn_details = base_ui_obj.check_element_text(
+            expected_text="Change connection details"
+        )
+        if conn_details:
+            logger.info("Click on Change Connection Details")
+            base_ui_obj.do_click(pvc_loc["connections-details"])
+        if ocp_version >= "4.9":
+            logger.info("Click on Create new KMS connection")
+            base_ui_obj.do_click(pvc_loc["new_kms"])
+        logger.info("Storage Class Service Name")
+        base_ui_obj.do_clear(pvc_loc["service-name"])
+        base_ui_obj.do_send_keys(pvc_loc["service-name"], service_name)
+        logger.info("Storage Class Address")
+        base_ui_obj.do_clear(pvc_loc["kms-address"])
+        base_ui_obj.do_send_keys(pvc_loc["kms-address"], kms_address)
+        logger.info("Storage Class Port")
+        base_ui_obj.do_clear(pvc_loc["kms-port"])
+        base_ui_obj.do_send_keys(pvc_loc["kms-port"], "8200")
+        logger.info("Click on Advanced Settings")
+        base_ui_obj.do_click(pvc_loc["advanced-settings"])
+        logger.info("Enter Backend Path")
+        base_ui_obj.do_clear(pvc_loc["backend-path"])
+        base_ui_obj.do_send_keys(pvc_loc["backend-path"], backend_path)
+        logger.info("Enter TLS Server Name")
+        base_ui_obj.do_clear(pvc_loc["tls-server-name"])
+        base_ui_obj.do_send_keys(pvc_loc["tls-server-name"], tls_server_name)
+        logger.info("Enter Vault Enterprise Namespace")
+        base_ui_obj.do_clear(pvc_loc["vault-enterprise-namespace"])
+        base_ui_obj.do_send_keys(pvc_loc["vault-enterprise-namespace"], vault_namespace)
+        logger.info("Selecting CA Certificate")
+        # print(os.path.abspath(constants.VAULT_CA_CERT_PEM))
+        # logger.info(os.path.abspath(constants.VAULT_CA_CERT_PEM))
+        # element = base_ui_obj.driver.find_element_by_xpath("//*[@id='modal-container']/div/div/div/form/div/div[2]/div/div/div[4]/div[2]/div/input")
+        # base_ui_obj.driver.execute_script("arguments[0].value='typed';",element)
+        base_ui_obj.send_keys(pvc_loc["browse-ca-certificate"], text=os.path.abspath(constants.VAULT_CA_CERT_PEM))
```

Comment:
I cannot find any reference for `base_ui_obj.send_keys` .
https://github.com/red-hat-storage/ocs-ci/blob/37613fb3cf8d076a8d80787258a4307d0633570f/ocs_ci/ocs/ui/base_ui.py#L330

Reply:
Why not doing something like  ? :
```python
file_input = driver.find_element(By.XPATH, "(//input[@type='file'])[1]")
file_input.send_keys('<your location>')
```

The `send_keys` function in selenium accepts text and also there is no such function on the `base_ui_obj` object of class `PageNavigator`.

Not too familiar with this code base but just cant find anything.
Also `do_send_keys` function wont work here due to that fact it waits for an element to be clickable, just guessing if you wanted to use that.

Reply:
@afreen23 Thanks, it's working now. :)